spec/system/hashtag_autocomplete_spec.rb

```diff
@@ -255,7 +255,9 @@ def visit_topic_and_initiate_autocomplete(initiation_text: "something #co", expe
 
     it "shows a default color and css class for the category icon square" do
       topic_page.visit_topic(topic, post_number: post_with_private_category.post_number)
-      expect(page).to have_css(".hashtag-cooked .hashtag-missing")
+      expect(page).to have_css(".hashtag-cooked .hashtag-category-badge")
+      css_class = ".hashtag-color--category--#{private_category.id}"
+      expect(find("#hashtag-css-generator", visible: false).text(:all)).not_to include(css_class)
```

Comment:
I think it'll be better to split this assertion out into another test because the description doesn't describe what this is asserting for.

Reply:
I cbanged the test a bit, does it make more sense now?